{"name": "Проверка авторизации без логина", "status": "failed", "statusDetails": {"message": "AssertionError: assert (400 == 400 and {'code': 400, 'message': 'Недостаточно данных для входа'} == {'message': 'Недостаточно данных для входа'}\n +  where 400 = <Response [400]>.status_code\n  \n  Common items:\n  \u001b[0m{\u001b[33m'\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m'\u001b[39;49;00m: \u001b[33m'\u001b[39;49;00m\u001b[33mНедостаточно данных для входа\u001b[39;49;00m\u001b[33m'\u001b[39;49;00m}\u001b[90m\u001b[39;49;00m\n  Left contains 1 more item:\n  \u001b[0m{\u001b[33m'\u001b[39;49;00m\u001b[33mcode\u001b[39;49;00m\u001b[33m'\u001b[39;49;00m: \u001b[94m400\u001b[39;49;00m}\u001b[90m\u001b[39;49;00m\n  \n  Full diff:\n  \u001b[0m\u001b[90m \u001b[39;49;00m {\u001b[90m\u001b[39;49;00m\n  \u001b[92m+     'code': 400,\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[90m \u001b[39;49;00m     'message': 'Недостаточно данных для входа',\u001b[90m\u001b[39;49;00m\n  \u001b[90m \u001b[39;49;00m }\u001b[90m\u001b[39;49;00m)", "trace": "self = <test_login_courier.TestLoginCourier object at 0x00000163191551D0>\nfor_auth_delete_cour = [{'address': 'клх Новгород Великий, ш. Заводское, д. 5/4 стр. 62, 311998', 'comment': 'Ныне снимать лапа передо тысяча...}, {'login': 'vlas78', 'password': 82788795}, {'login': 'vlas78', 'password': ''}, {'login': '', 'password': 82788795}]\n\n    @allure.title('Проверка авторизации без логина')\n    @allure.description(f'на ручке {Endpoint.login_courier_url}')\n    def test_login_courier_no_login(self, for_auth_delete_cour):\n        response = requests.post(f'{Endpoint.login_courier_url}', json=for_auth_delete_cour[3])\n>       assert response.status_code == 400 and response.json() == {\"message\":  \"Недостаточно данных для входа\"}\nE       AssertionError: assert (400 == 400 and {'code': 400, 'message': 'Недостаточно данных для входа'} == {'message': 'Недостаточно данных для входа'}\nE        +  where 400 = <Response [400]>.status_code\nE         \nE         Common items:\nE         \u001b[0m{\u001b[33m'\u001b[39;49;00m\u001b[33mmessage\u001b[39;49;00m\u001b[33m'\u001b[39;49;00m: \u001b[33m'\u001b[39;49;00m\u001b[33mНедостаточно данных для входа\u001b[39;49;00m\u001b[33m'\u001b[39;49;00m}\u001b[90m\u001b[39;49;00m\nE         Left contains 1 more item:\nE         \u001b[0m{\u001b[33m'\u001b[39;49;00m\u001b[33mcode\u001b[39;49;00m\u001b[33m'\u001b[39;49;00m: \u001b[94m400\u001b[39;49;00m}\u001b[90m\u001b[39;49;00m\nE         \nE         Full diff:\nE         \u001b[0m\u001b[90m \u001b[39;49;00m {\u001b[90m\u001b[39;49;00m\nE         \u001b[92m+     'code': 400,\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\nE         \u001b[90m \u001b[39;49;00m     'message': 'Недостаточно данных для входа',\u001b[90m\u001b[39;49;00m\nE         \u001b[90m \u001b[39;49;00m }\u001b[90m\u001b[39;49;00m)\n\ntests\\test_login_courier.py:17: AssertionError"}, "description": "на ручке https://qa-scooter.praktikum-services.ru/api/v1/courier/login", "start": 1754410542711, "stop": 1754410543325, "uuid": "12be3805-3e14-427f-8faa-19cc0444e35e", "historyId": "bdb9eb4630742c261284e3b04e46cfd4", "testCaseId": "bdb9eb4630742c261284e3b04e46cfd4", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_courier_no_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-GVQL230"}, {"name": "thread", "value": "14152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestLoginCourier"]}